srcpkgs/audit/template

```diff
@@ -0,0 +1,116 @@
+# Template file for 'audit'
+pkgname=audit
+version=2.8.4
+revision=1
+build_style=gnu-configure
+configure_args="--sbindir=/usr/bin --libdir=/usr/lib --enable-shared=audit
```

Comment:
`--sbindir=/usr/bin` is already defined by the build_style, it can be omitted here.